src/common/hooks/analytics/use-analytics.ts

```diff
@@ -11,11 +11,13 @@ function isIgnoredPath(path: string) {
   return IGNORED_PATH_REGEXPS.find(regexp => regexp.test(path));
 }
 
+export let tracker: any;
```

Comment:
Why are we declaring this and reassigning once? 🤔

Reply:
I initially added so we can use it outside react hooks. I removed it then to keep it encapsulated.